seatunnel-engine/seatunnel-engine-server/src/main/java/org/apache/seatunnel/engine/server/task/operation/source/AssignSplitOperation.java

```diff
@@ -51,7 +51,7 @@ public void run() throws Exception {
         SeaTunnelServer server = getService();
         RetryUtils.retryWithException(() -> {
             SourceSeaTunnelTask<?, SplitT> task =
-                server.getTaskExecutionService().getExecutionContext(taskID.getTaskGroupID()).getTaskGroup()
+                server.getSlotService().getSlotContext(taskID.getSlotID()).getTaskExecutionService().getExecutionContext(taskID.getTaskGroupID()).getTaskGroup()
                     .getTask(taskID.getTaskID());
```

Comment:
Can the encapsulation of the slot be improved? Hide details in TaskExecutionService, TaskID is unique for a job, slot-id and task-group-id can be hidden.